index.html

```diff
@@ -156,6 +156,40 @@ <h2 class="titles-about-me">About me</h2>
       </ul>
     </article>
   </section>
-    
+  <footer class="footer">
+    <p class="contact-par">I'm always interested in hearing about new projects,
+      so if you'd like to chat please get  in touch.</p>
+    <form action="https://formspree.io/f/mzbwkgww" method="post"class="form">
+      <input type="text"
+      placeholder="Full name"
+      required
+      maxlength="30"
+      name="name"
+      class="personal-info">
+      <input type="email"
+      placeholder="Email address"
+      required
+      maxlength="30"
+      name="Email"
+      class="personal-info">
+      <textarea name="comment" 
+      id="" 
+      cols="30" 
+      rows="10"
+      placeholder="Enter text here"
+      maxlength="500"
+      class="cmt-area"
+      required></textarea>
+      <button type="submit"class="button">Get in touch</button>
+    </form>
+    <ul class="socials">
+      <li><a href="#"><img src="tools/twitter.svg" alt=""></a></li>
+      <li><a href="#"><img src="tools/Linkedin icon.svg" alt=""></a></li>
+      <li><a href="#"><img src="tools/medium.svg" alt=""></a></li>
+      <li><a href="#"><img src="tools/github.svg" alt=""></a></li>
+      <li><a href="#"><img src="tools/angellist.svg" alt=""></a></li>
+    </ul>
+    <a href="#" class="dev-email">myemailhere@email.com</a>
```

Comment:
**_Good job so far, kindly fix these design issues to match the Figma template :_** 
- [ ] kindly add some space between the title and the form 
- [ ] kindly add some padding to your textarea input
- [ ] kindly align the content of the button in the center
- [ ] your social icons are justified to the center but not aligned, you can add ` align-items: center; `
- [ ] kindly add the background image as shown in the template



Figma Design           | Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/89282221/199449980-6cba314d-9866-4230-8bcc-fae8a2eb90ab.png) | ![image](https://user-images.githubusercontent.com/89282221/199449691-79a4d7af-a6fd-48ab-8aea-977870f7fb3e.png)